db/system_keyspace.cc

```diff
@@ -1463,7 +1463,47 @@ struct local_cache {
     system_keyspace::bootstrap_state _state;
 };
 
+future<> system_keyspace::peers_table_read_fixup() {
+    static bool done = false;
```

Comment:
`thread_local` or (better) we should assert that this is called on shard 0 only.

Reply:
done